dateparser/timezone_parser.py

```diff
@@ -44,11 +44,15 @@ def pop_tz_offset_from_string(date_string, as_offset=True):
         return date_string, None
 
 
+def is_word_match_any_tz(word):
```

Comment:
`is_word_match_any_tz` does not sound right to me. What about `word_is_tz`?

Reply:
Yep, it's better, fixed as it.